build.sbt

```diff
@@ -1045,12 +1050,18 @@ lazy val sbtClientProj = (project in file("client"))
     nativeImageOptions ++= Seq(
       "--no-fallback",
       s"--initialize-at-run-time=sbt.client",
+      // "The current machine does not support all of the following CPU features that are required by
+      // the image: [CX8, CMOV, FXSR, MMX, SSE, SSE2, SSE3, SSSE3, SSE4_1, SSE4_2, POPCNT, LZCNT, AVX,
+      // AVX2, BMI1, BMI2, FMA, F16C]."
+      "-march=compatibility",
       // "--verbose",
       "-H:IncludeResourceBundles=jline.console.completer.CandidateListCompletionHandler",
       "-H:+ReportExceptionStackTraces",
       "-H:-ParseRuntimeOptions",
       s"-H:Name=${target.value / "bin" / "sbtn"}",
-    ),
+    ) ++ (if (isLinux && isArmArchitecture)
+            Seq("-H:PageSize=65536") // Make sure binary runs on kernels with page size set to 4k, 16 and 64k
```

Comment:
Somehow "-H:PageSize=65536" disappeared from develop branch. Would like to double check whether we do need to add it to develop branch or remove it from both `1.10.x` and `develop` branch.